app/www/js/main.js

```diff
@@ -65,14 +95,61 @@ $(applicationCache)
     });
 
 // Utilities
-function addCounterValue(target, value){ 
+function addCounterValue(target, value) {
     const counter = $(target).attr("data-counter");
     const newValue = Number($(`input[data-counter='${counter}']`).val()) + value;
 
-    if (existing > 0)
+    if (newValue >= 0)
         $(`input[data-counter='${counter}']`).val(newValue);
 }
 
 function exists(object) {
-     return typeof object != undefined && object != null;
+    return typeof object != undefined && object != null;
+}
+
+function sendToDatabase() {
+    var jsonData = {
+        "Event Key" : $("#m-event-key").val(),
+        "Team Scouted" : Number($("#m-scouted").val()),
+        "Match Number" : Number($("#m-number").val()),
+        "Match Type" : $("#qual-match").val(),
+        "Match Type Number" : Number($("#m-sub-number").val()),
+        "Scout Team" : Number($("#m-scouting").val()),
+        "Scout Initials" : $("#m-scout").val(),
+
+        "Auto Bottom Rocket Panels" : Number($("#basic-addon1").val()),
+        "Auto Middle Rocket Panels" : Number($("#basic-addon2").val()),
+        "Auto Top Rocket Panels" : Number($("#basic-addon3").val()),
+        "Auto Bottom Rocket Cargo" : Number($("#basic-addon4").val()),
+        "Auto Middle Rocket Cargo" : Number($("#basic-addon5").val()),
+        "Auto Top Rocket Cargo" : Number($("#basic-addon6").val()),
+        "Auto Ship Panels" : Number($("#basic-addon7").val()),
+        "Auto Ship Cargo" : Number($("#basic-addon8").val()),
+
+        "Teleop Bottom Rocket Panels" : Number($("#basic-addon9").val()),
+        "Teleop Middle Rocket Panels" : Number($("#basic-addon10").val()),
+        "Teleop Top Rocket Panels" : Number($("#basic-addon11").val()),
+        "Teleop Bottom Rocket Cargo" : Number($("#basic-addon12").val()),
+        "Teleop Middle Rocket Cargo" : Number($("#basic-addon13").val()),
+        "Teleop Top Rocket Cargo" : Number($("#basic-addon14").val()),
+        "Teleop Ship Panels" : Number($("#basic-addon15").val()),
+        "Teleop Ship Cargo" : Number($("#basic-addon16").val()),
+        "Panel Ground Pickup" : $("#panel_ground_pickup").is(":checked") ? 1 : 0,
+        "Cargo Ground Pickup" : $("#cargo_ground_pickup").is(":checked") ? 1 : 0,
+
+        "Endgame Level Climbed" : Number($("#endgame_level_climbed").val()),
+        "Endgame Assist in Climbing" : $("#endgame_assist_in_climbing").is(":checked") ? 1 : 0,
+
+        "Reckless Driving" : $("#reckless-driving").is(":checked") ? 1 : 0,
+        "Not Present" : $("#not-present").is(":checked") ? 1 : 0,
+        "Disabled" : $("#disabled").is(":checked") ? 1 : 0,
+        "Robot Failure" : $("#robot-failure").is(":checked") ? 1 : 0,
+        "Top Heavy" : $("#top-heavy").is(":checked") ? 1 : 0,
+        "Foul" : $("#foul").is(":checked") ? 1 : 0,
+        "Card" : $("#card").is(":checked") ? 1 : 0
+    };
+    
+    // TODO: Clear Fields
```

Comment:
@ShashJar can you implement this? should be fairly straightforward, numeric fields default to 0, checkboxes to false, and text fields to ""